frontend/src/components/login/Login.tsx

```diff
@@ -21,36 +22,39 @@ export const Login: React.FunctionComponent = () => {
   }
 
   return (
-    <>
-      <div className="admin-message">
-        <h2>ONLY ACCESSIBLE TO ADMINS</h2>
+    <div>
```

Comment:
Not sure about this, consider making this outer-most <div> a blank tag <> as it's not got a className so we can't target it in css anyway? @TimLeach635 is that right or is a blank <div> ok?

```suggestion
    <>
```

Reply:
Agreed!

Reply:
Done